setup.py

```diff
@@ -70,7 +70,7 @@
             "databricks-sql-connector",
             "dbt-bigquery",
             "dbt-core",
-            "dbt-duckdb>=1.7.1",
+            "dbt-duckdb>=1.7.1,!=1.9.1",
```

Comment:
dbt-duckdb broke for python 3.8 after https://github.com/duckdb/dbt-duckdb/pull/482/files

Reply:
OTOH they don't test python 3.8, it seems like their support starts at ≥3.9. So we may have to fix the tests instead so they won't run for 3.8...

Reply:
Opened a ticket [upstream](https://github.com/duckdb/dbt-duckdb/issues/488) to track this.